app.test.js

```diff
@@ -77,6 +77,49 @@ describe('Server', () => {
       expect(result).toEqual(noMatch)
     });
   });
+
+  describe('POST /api/v1/projects', () => {
```

Comment:
I know we had started the post before and left it commented out, thanks for finishing it.